src/actions/BlockActions.js

```diff
@@ -298,12 +298,14 @@ export const updateBlockList = (lastBlock, startBlock, isLoadMore) => async (dis
 		}
 	});
 
-	const lastBlockStorage = blocksResult[blocksResult.length - 1];
-
-	if (lastBlockStorage) {
-		dispatch(BlockReducer.actions.set({ field: 'startTimestamp', value: 0 }));
+	const blockArray = Object.values(blocks.toJS());
+	if (blockArray) {
+		const lastBlockStorage = blockArray.reduce((acc, val) => (acc > moment.utc(val.timestamp).unix() ? acc : moment.utc(val.timestamp).unix()), 0);
```

Comment:
0 should not be start point, as default reduce use first element in array, better use this